File: Readio/src/main/java/ninegle/Readio/library/dto/book/AllLibraryBooksDto.java
package ninegle.Readio.library.dto.book;

import java.math.BigDecimal;
import java.time.LocalDate;

import lombok.Builder;

@Builder
public record AllLibraryBooksDto(Long bookId, String bookName, String bookImage, String bookIsbn, String bookEcn,
								 LocalDate bookPubDate, LocalDate bookUpdateAt, BigDecimal rating) {
}

File: Readio/src/main/java/ninegle/Readio/library/dto/book/AllLibraryBooksDto.kt
package ninegle.Readio.library.dto.book

import java.math.BigDecimal
import java.time.LocalDate


data class AllLibraryBooksDto(
    val bookId: Long?,
    val bookName: String?,
    val bookImage: String,
    val bookIsbn: String?,
    val bookEcn: String?,
    val bookPubDate: LocalDate,
    val bookUpdateAt: LocalDate?,
    val rating: BigDecimal?
)
